pkg/result/processors/fixer.go

```diff
@@ -57,7 +57,12 @@ func (f Fixer) Process(issues []result.Issue) []result.Issue {
 	for file, issuesToFix := range issuesToFixPerFile {
 		var err error
 		f.sw.TrackStage("all", func() {
-			err = f.fixIssuesInFile(file, issuesToFix)
+			fileWithoutPathPrefix := file
+
+			if f.cfg.Output.PathPrefix != "" {
+				fileWithoutPathPrefix = strings.Replace(fileWithoutPathPrefix, f.cfg.Output.PathPrefix+string(filepath.Separator), "", 1)
```

Comment:
```suggestion
				fileWithoutPathPrefix = strings.TrimPrefix(fileWithoutPathPrefix, f.cfg.Output.PathPrefix+string(filepath.Separator))
```

Reply:
Or we can use [`filepath.Base`](https://pkg.go.dev/path/filepath#Base)

Reply:
If I understood you correctly, I believe filepath.Base would not work as well here, as it's not guaranteed (as far as I can tell) that `pathPrefix` is just a single path element, it could have path separators in it as well.

Reply:
You could add tests to confirm or deny using `filepath.Base`.